README.md

```diff
@@ -88,59 +85,63 @@ contributors, is a good indication that the RFC is worth pursuing.
-* Fill in the RFC.
-  - The first line should be the [titled-cased version](https://en.wikipedia.org/wiki/Letter_case#Case_styles) of the RFC, prefixed with the `# ` (hash and space) followed by a blank line. This should be the only first-degree headline in the entire RFC!
-  - Put care into the details: RFCs that do not present convincing motivation, demonstrate understanding of the impact of the design, or are disingenuous about the drawbacks or alternatives tend to be poorly-received.
-  - If it makes sense, feel free to add further files in the same folder, giving further information, images or appendix, as long as they start with a non-number letter. For example, you may split the details of separate files per component touched.
-* Submit a pull request. As a pull request the RFC will receive design
-feedback from the larger community and the author should be prepared
-to revise it in response.
-* During triage, the pull request will either be closed (for RFCs
-that clearly will not be accepted), or assigned to a *shepherd*. The
-shepherd is a trusted developer who is familiar with the process, who
-will help to move the RFC forward and ensure that the right people
-see and review it.
-* Build consensus and integrate feedback. RFCs that have broad support
-are much more likely to make progress than those that don't receive
-any comments. The shepherd assigned to your RFC should help you get
-feedback from developers as well.
-* The shepherd may schedule meetings with the author and/or relevant
-stakeholders to discuss the issues in greater detail and in some
-cases the topic may be discussed at the larger [weekly meeting]. In
-either case a summary from the meeting will be posted back to the RFC
-pull request.
-* Once both proponents and opponents have clarified and defended
-positions and the conversation has settled, the shepherd will take it
-to the contributor team for a final decision.
-* Eventually, someone from the contributor team will either accept the RFC
-by merging the pull request, assigning the RFC a number (corresponding
-to the pull request number), at which point the RFC is "active", or
-reject it by closing the pull request.
-
-## The role of the shepherd
-[The role of the shepherd]: #the-role-of-the-shepherd
-
-During triage, every RFC will either be closed or assigned a shepherd.
-The role of the shepherd is to move the RFC through the process. This
-starts with simply reading the RFC in detail and providing initial
-feedback. The shepherd should also solicit feedback from people who
-are likely to have strong opinions about the RFC. Finally, when this
-feedback has been incorporated and the RFC seems to be in a steady
-state, the shepherd will bring it to the meeting. In general, the idea
-here is to "front-load" as much of the feedback as possible before the
-point where we actually reach a decision.
+- Fork the RFC repo [RFC repository](https://github.com/maidsafe/rfcs)
```

Comment:
Should probably be
```
Fork the [RFC repository](https://github.com/maidsafe/rfcs).
```

Reply:
agreed - damn I'm too keen my fingers are all thumbs!